com.gitlab.coringao.cavestory-nx.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "com.gitlab.coringao.cavestory-nx",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cavestory-nx",
+  "finish-args": [
+    "--device=all",
+    "--socket=pulseaudio",
+    "--socket=x11"
+  ],
+  "modules": [
+    {
+      "name": "cavestory-nx",
+      "no-make-install": true,
+      "buildsystem": "cmake-ninja",
+      "build-commands": [
+        "install -D cavestory-nx /app/bin/cavestory-nx",
+        "cp -R data /app/bin/data",
+        "install -Dm644 cavestory-nx.desktop /app/share/applications/com.gitlab.coringao.cavestory-nx.desktop",
+        "install -Dm644 data/cavestory-nx.png /app/share/icons/hicolor/512x512/apps/com.gitlab.coringao.cavestory-nx.png",
+        "install -Dm644 com.gitlab.coringao.cavestory-nx.appdata.xml /app/share/appdata/com.gitlab.coringao.cavestory-nx.appdata.xml"
```

Comment:
The `"${FLATPAK_ID}"` var can be useful here. The same applies to the `"${FLATPAK_DEST}"`.